tests/integration/test_workspace.py

```diff
@@ -21,6 +21,22 @@ def test_workspace_upload_download_notebooks(w, random):
     w.workspace.delete(notebook)
 
 
+def test_workspace_unzip_notebooks(w, random):
```

Comment:
Can we add a test to download the notebook 100 times and make sure that we don't exhaust our connection pool? We default to 20, so let's do 30 or so to be safe.

Reply:
I will add that one.